### DISCUSSION TITLE
Default Profile on Startup

### DISCUSSION THREAD

**Discussion by nicedevil007** [Eugeny/tabby#5417] [Q&A] [answered]
hey guys,

I'm looking for an option to close all tabs on closing tabby.
Each time I open tabby the default profile will be loaded and all other not closed tabs from the session before.

The default profile will also be loaded if the default profile is still open from the last session. Realy realy annoying.

Maybe I miss something in the config?

**Comment by Eugeny** [comment, marked as answer]
You can switch off "restore tabs" in settings - terminal.

**Reply by nicedevil007** [reply]
merci !!!! <3

**Comment by lawweiliang** [comment]
@nicedevil007  
Please close the discussion if the question was answered. 
It would be helpful to make the discussion section clean. 

Click `mark as answered` and `close the discussion`. 😄 

![image](https://user-images.githubusercontent.com/34936929/228098333-222ac3d4-4c39-4c26-a2a8-52906647e00e.png)
